community-operators/planetscale/planetscale-operator.v0.1.8.clusterserviceversion.yaml

```diff
@@ -0,0 +1,292 @@
nB2sL12qq4sivqf975tc15+4f5sqC9h4eG4TJkjqsoDniX1zIvbdQsRX Yr+sSsj6uT9ek39sCNwj9M8kthPdMQY4MXKOcysu8hEsIgeyECVwbA7ck2PgviUEbVcEWNvAayAf QMQqpjIBhIhlAOgLJDr7CheiBeGvQcbJILqNsoAN9XlTUQBrn84qceyRxXYvogDq3Nfm5P52LswW u7Jajz+Hcl6F889Dwr1L1+ErZL+XIwtzzlj4JHLDh1zC1VICaw1kJ0sLjpcYRM4r0O2H3jlNNJg4 UJ6AELENACGE2Mpx4CIed/kpmTajANrIBdCrpQArfLCpbN7pdtFc1rAombbrehYxjAxTLfeuyuG5 ObhZJfrn/sy/fn6/sNdM/vaaBBCMoHnpYXa/ARDKG4acAsfPEb/+V7b2c2ntsoD1ogBWMgHqvMBi mgDKBSBkAAghhDgR/quNNnpRfi3yAWtmJVKnJOzIJtbSpy63tXZUBvcsI/FB7JdzX/1bld4rfZWY 72ziPZPpthb331dn0gGU4Kp7K5gB/jipYDAFTLyIkBR5A7r0vFm3reYKogghA0AIIXo7ljNmtcUl 6+YD6HB5p06q/6bX9DcZBRDdODj5Yyulutpa23/KxDsuCetD9n039ZQzKKf+OOP+IgP/6fXcVoJf rGoILO69e4wkKkPAVpEBp3MIUBkHSXIBXPGhpLkAIhgVySO/5AoIGQBCCLFtAzbTflaJPg9AGhbW rZQF7Po1XdIE2Njb+FRIvnfVTGz1b1kJfjf1uNPZ+k8dszGABL9IZQgsODaiKlPOnqoycFyKsEbe gMbKAnbswdBWKVshZAAIIcQGDM4W8ZhXpQKvOVxofIDRdi6CnngOvdhhU0sBunrNa8zaLUS/m0M5 JQj9mT8O63fze4SYuUCYCdHUvX2KYFaFe5XTkQFLmAHNf33XyAWQYu9rzu6nXrIkhAwAIYTokuhf afTfQCr0tssCtpFheZ19NlQW8Dg6d9XwfJb7XLqygBH3s8T5JEsIeDphH2H9/mzqKadV4r5K9Ht/ TzZ2IfpgCvhgXnlO7l1rq9KD2flG9XE01JLPmKVfYxe22VBCQB/52aKlAELIABBCbOk4y5pGBXyb Y4k2Zsh7VRawowO9dCZAg1N1sfrCnBzbYh3/bOqZT6pw/vKkRJ+BEEKdaWZf9NsMWNy/rgxLWCjB VrkDFlUFTn9vO/H1bTDEXxpdCBkAQgix3KAqSurlHtSEa9l16E1ZwA53bS8GuCkqAphTYc9uUYov zIrOjsK/zi1qsZ+YBErWJzbytXU6MmCRpHJ+KolgdmKSGWpEQ6UoC8g67bX3wJSZIGQACCHEJg6e 6gj/TYkCWGdmSCOiDe6fCyoCNHVeF4T1H6/jL8M6fjcPYf2zo/DvvU2YM2U6hdiS587CD3BQOo+Z nUQELMyANpIH9uZZq1wAQgaAEEJs9mhpqdx+Kw066i+C3raygI1HATSUC+D4I4lzAZw5tKbKAia4 XpedsrFhA+9gfgTlxDM/ztZfbWSJVzNdiA3DA34ecgac5Ao4Vc7Wr/79XSkK4Ir2WOWZFlu0ywQQ MgCEEGLDZH+lFjohGvpuAvRqhzSfC6DJ1SBNmQCpzul0zk0Lfg7Tu55ycipr/6z63lZCRgix/HfL u7BMwMzBWI89nTiwyUdqg8n2kj+CZQIIGQBCCNHxcVBmjsPe9Q6n9VwCTVcFWHlmvumu2bSkBRfN qJ0O03dQltUsfzXb71yV4MwH0W81MhFibSMAwveqdGDmHixkWVU+44IlOEtXBfCRHzANJg5c6pkl vS9kAAghRE/GPWZRGyzii70rI4GIIn4b8gE0XopwQ8R8tCiA0+v552F2v5zAfOKZH3l8ebJGWSX6 hEj4rKBaFlXCvDyVONCG793SuQJWeTC1WRFASwGEkAEghNjcAY45E1fcUbmZIKlgr67RBpcFTBee H/nyN1AW8DgBmavE/iQI/9lhEP3Y6qtqTv4VQjSHd1CWHmMNJg8mgLXL5U5Z6VmnWXYhZAAIIURj wj9mFAAskRBweROgZh646D5FG7kAWikLuGFLAdYyG5Y0AWrPfJmTXBvzqWd+UM32T0Pm/sUMv9bz C9Gd96f3HjetxP9i6Y01ocTgys+SBsoCXvLyjB0FINdCyAAQQoiujF1sw+X8urmz5g8xwjKElU2A HlyRrRkbLgS/D7OGbhbK9M0Pqsz98zDLaLSeX4huf5XNSdLA8J312Hzd9+saJkDHHszS+0IGgBBC dEJ4mJU+F+UtniCMv5WcBFUbvcoF0PR+N7UiwJIHf26OsFPl+soZzI88s7thlt/78PvFdlrTL0SP 3qtU3+ESnPOhlKA1x9/5td5ZdT7U5tKCuhEIQsgAEEKI1MK/L3qygXwAfZ6WaCEKYBMT+6VbcnD/ aN9mleifQDn1zA5hfgiuPCm1YZDoF2Ij8OBKoPSYzGCz+3MExDYBGilBuoWvWyFkAAghJPw7+mJv 8zhUEaDnO0y0L1MJ/3IO01swmzjmBzCfVmI/k+AXYuO9gNJTluArIwC77POm3eQrySsCCCEDQAgh 4osPTOS3bqNlAesfeONLAdosC7hqFMA6aznNclmu193ncX6qVSsCQNJKAhd+5nRWfm+YHbkw038U svn7slrTr0R+Qmwd5dyfyuthzj5CLnxu1SsLGDsKQCaAEDIAhBB9Ef8pZxU7nAuglTb6uBShB2UB m+qftQfMpyJsvAuz/bMjz+yuYz4N64Hx1Zp+jRaE2N73cmWg+jl4Vy0NsNXv6z4EG8zMnzwfgBAy AIQQYj01c264f5NJ1XrQ6kZd7z6VBezMGv0I+1mU5TMhmV9ZJfObHoZZPu9OthFCiNPPjmAEeBwh GsBm1XMV4s/a67UshAwAIcTmDSYg7Tr/ZQcAbWbjb3Vssk5FgJ7eclubD2CR0K+E+aFnPoHpXcf8 8NT3UMJfCLEkbu7xz
```

Comment:
This value should be the semver of the CSV. eg: `0.1.8`

Reply:
GitHub previews doesn't quite work with larger files (or at least whenever a big enough icon is used). My comment is in reference to line 131.

Reply:
@SamiSousa: which value exactly?

Reply:
I see, thank you.